source/runbooks/dr-process.html.md.erb

```diff
@@ -0,0 +1,361 @@
+---
+owner_slack: "#modernisation-platform"
+title: Disaster Recovery Process
+last_reviewed_on: 2024-01-24
+review_in: 6 months
+---
+
+
+# <%= current_page.data.title %>
+
+
+This page is designed to help give direction in the event of a DR scenario.
+
+
+It covers permanent loss of all infrastructure, temporary loss of all infrastructure, and loss of a single account.
+
+
+## Diagnose
+
+
+To find out the severity of the situation, check the AWS status page [here](https://health.aws.amazon.com/health/status ).
+
+
+Regardless of which type of disaster recovery situation there is, you should contact AWS **immediately**, understanding if this is an outage for 1 hour or 1 month will dictate the next steps to take.
+
+
+- If the issue is across all accounts, login to the Modernisation Platform account, and raise the highest level severity ticket, whilst also enganging with our AWS representative in the `#ext-aws` channel.
+
+
+- If this issue is in one account, login to that account, and raise the highest level severity ticket if production, and also enganging with our AWS representative in this channel **fill in**
```

Comment:
do you mean to have "**fill in here** "?

Reply:
was it suppose to be this?
`#ext-aws`